Packs/CiscoStealthwatch/Integrations/CiscoStealthwatch/test_data/list_security_events_results.json

```diff
@@ -0,0 +1 @@
+{"data":{"results":[{"id":5223,"domainId":102,"deviceId":121,"securityEventType":310,"firstActiveTime":"2021-02-04T13:20:38.000+0000","lastActiveTime":"2021-02-04T13:20:38.000+0000","source":{"ipAddress":"192.168.91.6","port":0,"protocol":"udp","tags":[{"name":"Catch All","id":65534}]},"target":{"ipAddress":"52.109.8.21","port":137,"protocol":"udp","tags":[{"name":"United States","id":61627}]},"details":[{"key":"source_host@username","value":""},{"key":"source_host@policy_id","value":"1"},{"key":"source_host@mac_address","value":""},{"key":"target_host@username","value":""},{"key":"target_host@policy_id","value":"0"},{"key":"target_host@mac_address","value":""},{"key":"category_points@high-concern-index","value":"162"},{"key":"category_points@high-target-index","value":"162"},{"key":"category_points@high-recon-index","value":"162"},{"key":"baseline@baseline","value":"0"},{"key":"baseline@threshold","value":"0"},{"key":"baseline@current_value","value":"0"},{"key":"baseline@tolerance","value":"0"},{"key":"flow@protocol","value":"17"},{"key":"flow@service","value":"16"},{"key":"flow@source_port","value":"0"},{"key":"flow@target_port","value":"137"},{"key":"flow@event_port","value":"137"},{"key":"flow@flow_id","value":"0"},{"key":"flow@source_is_server","value":"false"},{"key":"targetIPAddress","value":"52.109.8.21"},{"key":"points","value":"162"}],"hitCount":1},{"id":4686,"domainId":102,"deviceId":121,"securityEventType":310,"firstActiveTime":"2021-02-04T10:44:30.000+0000","lastActiveTime":"2021-02-04T13:20:38.000+0000","source":{"ipAddress":"172.16.100.4","port":0,"protocol":"tcp","tags":[{"name":"Catch All","id":65534}]},"target":{"ipAddress":"192.168.91.12","port":445,"protocol":"tcp","tags":[{"name":"Catch All","id":65534}]},"details":[{"key":"source_host@username","value":""},{"key":"source_host@policy_id","value":"1"},{"key":"source_host@mac_address","value":""},{"key":"target_host@username","value":""},{"key":"target_host@policy_id","value":"1"},{"key":"target_host@mac_address","value":""},{"key":"category_points@high-concern-index","value":"248994"},{"key":"category_points@high-target-index","value":"248994"},{"key":"category_points@high-recon-index","value":"248994"},{"key":"baseline@baseline","value":"0"},{"key":"baseline@threshold","value":"0"},{"key":"baseline@current_value","value":"0"},{"key":"baseline@tolerance","value":"0"},{"key":"flow@protocol","value":"6"},{"key":"flow@service","value":"29"},{"key":"flow@source_port","value":"0"},{"key":"flow@target_port","value":"445"},{"key":"flow@event_port","value":"445"},{"key":"flow@flow_id","value":"0"},{"key":"flow@source_is_server","value":"false"},{"key":"targetIPAddress","value":"192.168.91.12"},{"key":"points","value":"248994"}],"hitCount":1537}]}}
```

Comment:
Are you sure there are no secrets here?

Reply:
I removed IP addresses and the company name just to be sure, thanks!